pkg/minikube/machine/cache_images.go

```diff
@@ -17,18 +17,19 @@ limitations under the License.
 package machine
```

Comment:
I supposed this never worked, but we never cared about checking the error since it was done on a best-effort basis.  But deleting the temp image tarballs in the VM requires sudo

```diff
diff --git a/pkg/minikube/machine/cache_images.go b/pkg/minikube/machine/cache_images.go
index ec26c0230..63cf4e40a 100644
--- a/pkg/minikube/machine/cache_images.go
+++ b/pkg/minikube/machine/cache_images.go
@@ -17,7 +17,6 @@ limitations under the License.
 package machine

 import (
-       "golang.org/x/sync/errgroup"
        "io/ioutil"
        "os"
        "os/exec"
@@ -25,6 +24,8 @@ import (
        "runtime"
        "strings"

+       "golang.org/x/sync/errgroup"
+
        "k8s.io/minikube/pkg/minikube/assets"
        "k8s.io/minikube/pkg/minikube/bootstrapper"
        "k8s.io/minikube/pkg/minikube/config"
@@ -212,7 +213,7 @@ func LoadFromCacheBlocking(cmd bootstrapper.CommandRunner, src string) error {
                return errors.Wrapf(err, "loading docker image: %s", dst)
        }

-       if err := cmd.Run("rm -rf " + dst); err != nil {
+       if err := cmd.Run("sudo rm -rf " + dst); err != nil {
                return errors.Wrap(err, "deleting temp docker image location")
        }
```